Vagrantfile

```diff
@@ -0,0 +1,35 @@
+Vagrant.configure("2") do |config|
+  config.vm.box = "hashicorp/bionic64"
+  config.vm.provision "shell", privileged: false, inline: <<-SHELL
+    sudo apt-get update
+
+    #install pyenv prereqs
+    sudo apt-get install -y build-essential libssl-dev zlib1g-dev libbz2-dev \
+    libreadline-dev libsqlite3-dev wget curl llvm libncurses5-dev libncursesw5-dev \
+    xz-utils tk-dev libffi-dev liblzma-dev python-openssl git
+
+    #install pyenv
+    git clone https://github.com/pyenv/pyenv.git ~/.pyenv
+    echo 'export PYENV_ROOT="$HOME/.pyenv"' >> ~/.bash_profile
+    echo 'export PATH="$PYENV_ROOT/bin:$PATH"' >> ~/.bash_profile
+    echo -e 'if command -v pyenv 1>/dev/null 2>&1; then\n  eval "$(pyenv init -)"\nfi' >> ~/.bash_profile
+    pyenv install 2.7.8
```

Comment:
Is there any reason to install python 2.7.8?

Separately, when I run vagrant up, it actually fails to run pyenv the first time because although you've set it to be added to the path the next time the shell opens, it hasn't been added in the current shell. 
A solution to that would be to reload the environment variables set in your `bash_profile` by running `source ~/.bash_profile` before trying to execute pyenv